src/org/scada_lts/utils/SystemSettingsUtils.java

```diff
@@ -380,6 +381,16 @@ public static int getThreadsNameAdditionalLength() {
         }
     }
 
+    public static int getDataPointExtendedNameLengthInReportsLimit() {
+        try {
+            String config = ScadaConfig.getInstance().getConf().getProperty(DATA_POINT_EXTENDED_NAME_LENGTH_IN_REPORTS_LIMIT, "64");
```

Comment:
In env.properties you need to set 64, and here you can set 65 to know what the source of the default value is and whether something went wrong while reading the properties from the file.